www/index.html

```diff
@@ -1,11 +1,18 @@
-<!DOCTYPE html>
-<html>
-<head lang="en">
-    <meta charset="UTF-8">
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
     <title>Joomla! Update Channel</title>
-</head>
-<body>
-<h1>Joomla! Update Channel</h1>
-<p>This server provides Joomla! CMS updates for core 2.5 to 4.x installations as well as the listing of official translation packages.</p>
-</body>
+  </head>
+  <body>
+    <h1>Joomla! Update Channel</h1>
+    <p>
+      This server provides Joomla! CMS updates for core 2.5 to 5.x installations
```

Comment:
```suggestion
      This server provides Joomla! CMS updates for core 2.5 to 6.x installations
```

Sorry my fault, der server provides joomla 6 but not this repository.